core-bundle/src/EventListener/DataContainer/DefaultOperationsListener.php

```diff
@@ -177,6 +181,7 @@ private function getDefaults(string $table): array
         if ($canDelete) {
             $operations['delete'] = [
                 'href' => 'act=delete',
+                'method' => 'DELETE',
```

Comment:
According to @fritzmg's https://github.com/contao/contao/issues/2715#issuecomment-2979662827, this will not work in Contao.

Reply:
it adds a hidden `_method` field to the form, which is ignored by our current framework. It doesn't really matter though, I just thought this is more correct (also for a future Ajax call or thelike).

Reply:
Using `DELETE` as an actual method will work. What will not work is using `POST` with a `_method` override. But I am not sure whether this matters here?